### DISCUSSION TITLE
LegacyComboCounter changes size based on anchor.

### DISCUSSION THREAD

**Discussion by zachmanthethird** [ppy/osu#15969] [Q&A] [answered]
Version 2021.1206.0
https://user-images.githubusercontent.com/74262407/144950328-a91192d6-4df5-4537-adf2-36a941ebe25d.mp4
Changing the anchor of the LegacyComboCounter causes it to exhibit strange behaviors. The center anchor causes the sprite to dramatically increase in height and width. The upper-right anchor causes the sprite to have no size and thus unmovable.

**Comment by smoogipoo** [comment, marked as answer]
https://github.com/ppy/osu/issues/15973
